gpio.py

```diff
@@ -17,9 +17,10 @@ class PinState(object):
         value: the file pointer to set/read value of pin.
         direction: the file pointer to set/read direction of the pin.

```

Comment:
Add docs for active_low, including default behavior of `None` value

Reply:
See my comment below, I don't believe this value can be `None`, therefore it doesn't need docs for that. I added docs about the new `None` value of `active_low` to `setup()`

Reply:
You're right, I didn't fully understand the code path.